Algorithm.Framework/Risk/MaximumDrawdownPercentPortfolio.py

```diff
@@ -50,6 +50,7 @@ def ManageRisk(self, algorithm, targets):
 
         # Update trailing high value if in trailing mode
         if self.isTrailing and self.portfolioHigh < currentValue:
+            self.initialised = False # reset the trailing high value for restart investing on next rebalcing period
```

Comment:
Hey! Thanks for your contribution.
The effect of `self.initialised = False` seems to be resetting the `self.portfolioHigh` which we are already setting here too? Not sure _why_ this change is required and the effect it's having

>If trailing stop is True, the algo reached liquidation requirement will always be stuck as the 'high' value of portfolio is always higher than current value by preset drawdown percent. It will never restart investing on next rebalancing period and beyond.

^ From the reported issue. It should restart investing once it's out of the drawdown?
I think we should add some unit test (see MaximumDrawdownPercentPerSecurityTests) reproducing the underlying reported issue so it's clearer and we make sure the expected behavior is covered

Reply:
ooops sorry I added it at a wrong line. Should be the if condition below

Reply:
plz see comment on below and review

Reply:
Thanks, it makes sense! 👍 We should update the C# implementation too `MaximumDrawdownPercentPortfolio.cs` I'll make a GH issue for adding some tests for this